src/dotnet-roslyn-tools/Commands/CreateReleaseTagsCommand.cs

```diff
@@ -32,7 +38,9 @@ public async Task<int> InvokeAsync(InvocationContext context)
             var logger = context.SetupLogging();
             var settings = context.ParseResult.LoadSettings(logger);
 
-            return await CreateReleaseTags.CreateReleaseTags.CreateReleaseTagsAsync(settings, logger);
+            var product = context.ParseResult.GetValueForOption(ProductOption)!;
```

Comment:
Why the ! here?

Reply:
`GetValueForOption` is annotated as `MayBeNull`, but we have an allow list and a default for the option, so it won't be. All of the commands do this.